sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -79,6 +91,17 @@
                 offline_store_creator=RedshiftDataSourceCreator,
                 online_store=REDIS_CONFIG,
             ),
+            # Snowflake configurations
+            IntegrationTestRepoConfig(
```

Comment:
as discussed, we can remove the snowflake online store here.

Also we should modify the test_universal_types to exclude snowflake for list features